pfb-analysis/scripts/setup_database.sh

```diff
@@ -7,15 +7,32 @@ NB_POSTGRESQL_USER=gis
 NB_POSTGRESQL_PASSWORD=gis
 
 # Set defaults for overridable configuration params
-PFB_WORK_MEM="${PFB_WORK_MEM:-2048MB}"
 PFB_CHECKPOINT_COMPLETION="${PFB_CHECKPOINT_COMPLETION:-0.8}"
-PFB_MAX_WAL_SIZE="${PFB_MAX_WAL_SIZE:-2GB}"
+# Disable to improve performance, if disabled, data loss occurs on server crash (ok for analysis)
+PFB_FSYNC="${PFB_FSYNC:-off}"
```

Comment:
I recently played a bunch of games to try and get the `pg_restore` times for Cicero down and settled on the following options, which made a significant difference compared to the defaults:

- `autovacuum off`
- `checkpoint_segments 256`
- `checkpoint_timeout 1800`
- `maintenance_work_mem 1048576` (1GB; might need to tweak given VM resources)
- `synchronous_commit off`
- `wal_buffers 8192`

I'd probably leave `fsync` on and try these settings to see how much better than is from the baseline.